MaxText/llama_or_mistral_ckpt.py

```diff
@@ -30,25 +32,30 @@
 # pylint: disable=g-line-too-long
 import argparse
 import pathlib
+import os
+import sys
+import gc
+import re
+import logging
+from dataclasses import dataclass
 
-import numpy as np
+os.environ["JAX_PLATFORMS"] = "cpu"
```

Comment:
I think we shouldn't have this, it can be an option to run on whatever hardware, and user can set this env var before running the script if necessary